crates/sui-node/src/lib.rs

```diff
@@ -326,6 +326,7 @@ impl SuiNode {
                         .prune_and_compact_before_upload
                         .unwrap_or(true),
                     config.indirect_objects_threshold,
+                    config.authority_store_pruning_config.clone(),
```

Comment:
let's not forget to update pruning configs on current snapshot nodes to enable pruning for the main db